onboarding/forms.py

```diff
@@ -82,6 +82,15 @@ class OnboardingStep1Form(AptNumberWithConfirmationForm, AddressAndBoroughFormMi
 
     last_name = forms.CharField(max_length=150)
 
+    preferred_first_name = forms.CharField(
+        max_length=150,
+        required=False,
+        help_text=(
+            "The first name Justfix will call the user by. Optional. May be different from "
+            "their legal first name."
+        ),
+    )
+
```

Comment:
Hmm, so you might actually need to make a duplicate version of this called `OnboardingStep1FormV2` or somesuch--it's possible that due to the way that these forms are translated to GraphQL, old versions of the front-end that don't specify any value for preferred first name (as opposed to specifying an empty string for it) will make the GraphQL endpoint explode.  😞